brandingbrand/flagship
# fix(fscomponents): Fix updating button title

The title state would not update if dynamicTitleStates wasn't being used

TEST
----------------
- build storybook
- go to the Button component
- should be able to edit button titles and see your change reflected
